app/main.py

```diff
@@ -1,3 +1,81 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __add__(self, other: int) -> Distance:
```

Comment:
The type annotation for `other` should include `Distance`, `int`, and `float` since the method handles these types. Consider changing the signature to `def __add__(self, other: Union[Distance, int, float]) -> Distance:`.